src/_guides/language/extension-methods.md

```diff
@@ -212,14 +213,28 @@ extension NumberParsing on String {
 ```
 <div class="prettify-filename">lib/string_apis.dart</div>
 
-To create a local extension that's visible only in
-the library where it's declared,
-either omit the extension name or give it a name
-that starts with an underscore (`_`).
-
-The members of the extension can be methods, getters, setters, operators.
+The members of an extension can be methods, getters, setters, or operators.
 Extensions can also have static fields and static helper methods.
 
+### Unnamed extensions
+
+You can optionally omit the name when declaring an extension.
+An unnamed extension is visible only
+in the library where it's declared.
+You also can't apply it explicitly
+to resolve [API conflicts](#api-conflicts).
```

Comment:
Thanks for feedback. I've decided to rewrite the API conflict sentence to provide some extra context.

I needed to keep the word "explicitly" since its a specific type of usage mentioned above, in explicitly using the name to apply an extension rather than applying it directly through the underlying type. 

Let me know what you think of the adjusted text.